src/Main.hs

```diff
@@ -8,12 +8,13 @@ import Config
 import Control.Concurrent (forkIO)
 import Control.Concurrent.Chan (newChan, readChan)
 import Control.Monad (unless)
-import Data.Traversable (forM)
 import Language.C (parseCFile)
 import Language.C.Data.Ident (Ident(Ident))
+import qualified Language.C.Parser as CParser
 import Language.C.Syntax.AST (CTranslUnit)
 import Language.C.System.GCC (newGCC)
 import System.Exit (exitFailure)
+import qualified System.FilePath as FP
```

Comment:
I try not to abbreviate import aliases, usually taking the last name part (`FilePath`). But it seems like you just need `takeExtension` here.